Index: pixman-0.30.0/Makefile.am
===================================================================
--- pixman-0.30.0.orig/Makefile.am	2013-04-18 00:03:32.000000000 -0500
+++ pixman-0.30.0/Makefile.am	2014-01-10 16:54:05.146255371 -0600
@@ -1,4 +1,12 @@
-SUBDIRS = pixman demos test
+SUBDIRS = pixman
+
+if ENABLE_DEMOS
+SUBDIRS += demos
+endif
+
+if ENABLE_TESTS
+SUBDIRS += test
+endif
 
 pkgconfigdir=$(libdir)/pkgconfig
 pkgconfig_DATA=pixman-1.pc
Index: pixman-0.30.0/configure.ac
===================================================================
--- pixman-0.30.0.orig/configure.ac	2014-01-10 16:51:50.482252913 -0600
+++ pixman-0.30.0/configure.ac	2014-01-10 16:52:32.178253674 -0600
@@ -268,6 +268,31 @@
 ])
 
 dnl ===========================================================================
+dnl Enable tests/demos
+
+AC_ARG_ENABLE(tests,
+   [AC_HELP_STRING([--disable-tests],
+                   [disable building the test suite])],
+   [enable_tests=$enableval], [enable_tests=yes])
+
+if test $enable_tests = yes ; then
+   AC_DEFINE(ENABLE_TESTS, 1, "Enable building test suite")
+fi
+
+AM_CONDITIONAL(ENABLE_TESTS, test $enable_tests = yes)
+
+AC_ARG_ENABLE(demos,
+   [AC_HELP_STRING([--disable-demos],
+                   [disable building the demos])],
+   [enable_demos=$enableval], [enable_demos=yes])
+
+if test $enable_demos = yes ; then
+   AC_DEFINE(ENABLE_DEMOS, 1, "Enable building demos")
+fi
+
+AM_CONDITIONAL(ENABLE_DEMOS, test $enable_demos = yes)
+
+dnl ===========================================================================
 dnl Check for Loongson Multimedia Instructions
 
 if test "x$LS_CFLAGS" = "x" ; then
